v3/setup/images-control.sh

```diff
@@ -12,18 +12,18 @@ if [ "$(etcdctl get images-control-bootstrapped)" == "true" ]; then
 fi
 etcdctl set images-control-bootstrapped true
 
-etcdctl set /images/chronos      "mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/fd           "behance/flight-director:latest"
-etcdctl set /images/marathon     "mesosphere/marathon:v0.15.1"
-etcdctl set /images/mesos-master "mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
-etcdctl set /images/zk-exhibitor "behance/docker-zk-exhibitor:latest"
-etcdctl set /images/cfn-signal   "behance/docker-cfn-bootstrap:latest"
+etcdctl set /images/chronos      	"mesosphere/chronos:chronos-2.4.0-0.1.20150828104228.ubuntu1404-mesos-0.27.0-0.2.190.ubuntu1404"
+etcdctl set /images/flight-director "behance/flight-director:latest"
+etcdctl set /images/marathon     	"mesosphere/marathon:v0.15.1"
+etcdctl set /images/mesos-master 	"mesosphere/mesos-master:0.27.0-0.2.190.ubuntu1404"
```

Comment:
Can we make these the latest tested `mesosphere/mesos-master:0.28.1` and `mesosphere/marathon:v1.1.1`